packages/fast-components-react-base/src/index.ts

```diff
@@ -114,7 +114,11 @@ import Typography from "./typography";
 export { Typography };
 export * from "./typography";
 
-export { WindowWithIntersectionObserver, WindowWithResizeObserver } from "./utilities";
+export {
+    WindowWithIntersectionObserver,
+    WindowWithResizeObserver,
+    extractHtmlElement,
```

Comment:
We do have a `@microsoft/fast-react-utilities` package this may be better suited for.

Reply:
Oh yeah, I didn't want to put it in generic web utilities and missed that we had a react specific one.  I'll move it there.

Reply:
Moved, and also changed the ref prop to "any" as you noted in the other PR:
export function extractHtmlElement(
    sourceRef: React.RefObject<any> | HTMLElement
): HTMLElement | null {...